vaadin-tabs-flow-parent/vaadin-tabs-flow/src/main/java/com/vaadin/flow/component/tabs/TabSheet.java

```diff
@@ -281,6 +281,15 @@ public Tab getTab(Component content) {
                 .map(Map.Entry::getKey).findFirst().orElse(null);
     }
 
+    /**
+     * Returns the {@link Tabs} in this tab sheet.
+     *
+     * @return the tabs
+     */
+    public Tabs getTabs() {
```

Comment:
See https://github.com/vaadin/flow-components/issues/5232#issuecomment-1700658159

Reply:
Thanks. Got it. I will close this PR till we decide on what to do with the issue.